open-api/src/testFixtures/java/org/apache/iceberg/rest/RESTCatalogServer.java

```diff
@@ -37,12 +38,19 @@
 public class RESTCatalogServer {
   private static final Logger LOG = LoggerFactory.getLogger(RESTCatalogServer.class);
 
-  static final String REST_PORT = "rest.port";
+  public static final String REST_PORT = "rest.port";
   static final int REST_PORT_DEFAULT = 8181;
 
   private Server httpServer;
+  private final Map<String, String> config;
 
-  RESTCatalogServer() {}
+  public RESTCatalogServer() {
```

Comment:
```suggestion
 RESTCatalogServer() {
```

Reply:
I don't think this needs to be public anymore, so we can just go back to how it was before